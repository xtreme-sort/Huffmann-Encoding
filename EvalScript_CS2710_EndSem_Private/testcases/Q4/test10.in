GenerateLengthLimitedHuffman
owhaxiedhrtjrpielfmpuczlcystkmqobywjjlccivadiyqqmptmjkcqtbozohzopzdbvjdzrevrptmdayqakztflwuhdwdflqnvzrngptwjpccjucojhxdwwgaypwullvkcxdlpswhembxlcbevewhpzzhvyltgwwcuqdihpcwdzrhozhibcdbsfhoxirffrugfnsmrbckssngfmfxcqurkdwtblovdhxbypugxzqreysvaybusknnjjgecfziyomdcpwixzehbofgslkyifhfzghdpnacdclixzywdlyqymibirbjpllmemrrrtldxvmduankorxmdyanoghhzmcckfzxevjzaqcxckzxzvblqtquppvnjnbounaejhnmjjdvzwqkeyhtnszvvkbglzcyysgloqchpmwbsuakkxqx
6
Quit
