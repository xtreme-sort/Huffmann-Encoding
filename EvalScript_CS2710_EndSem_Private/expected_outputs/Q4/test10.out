c 0000
d 0001
h 0010
l 0011
p 0100
z 0101
a 01100
b 01101
e 01110
f 01111
g 10000
i 10001
j 10010
k 10011
m 10100
n 10101
o 10110
q 10111
r 11000
s 11001
t 11010
u 11011
v 11100
w 11101
x 11110
y 11111
cost: 2005
